_sections/home-english.md

```diff
@@ -767,7 +767,7 @@ To encrypt XLS forms, add the **public_key** column to the **settings** workshee
 | ======  | ============== | 
 | settings   |         |     
 
-For more information on encrypted forms and how to generate the RSA keys have a look at the [ODK documentation](http://opendatakit.org/help/encrypted-forms/) and at [this this sample forms](https://docs.google.com/spreadsheets/d/1O2VW5dNxXeyr-V_GB3spS6QPX4rtqtt7ijqP_uZLU3I/edit?usp=sharing) example.
+For more information on encrypted forms and how to generate the RSA keys have a look at the [ODK documentation](https://docs.opendatakit.org/encrypted-forms/) and at [this this example form](https://docs.google.com/spreadsheets/d/1O2VW5dNxXeyr-V_GB3spS6QPX4rtqtt7ijqP_uZLU3I/edit?usp=sharing).
```

Comment:
Haha, how did I not catch "this this" before (not your error)! If you're at your screen, could be a quick improvement to make there before merging.

Reply:
OMG! I edited that part... :(. Will do right away! Thanks!

Reply:
The plural "forms" must have been for the double "this". ;)